src/Nethermind/Nethermind.Runner/Ethereum/Steps/SetupHive.cs

```diff
@@ -23,7 +23,7 @@
 
 namespace Nethermind.Runner.Ethereum.Steps
 {
-    [RunnerStepDependencies(typeof(SetupKeyStore))]
+    [RunnerStepDependencies(typeof(SetupKeyStore), typeof(LoadGenesisBlock))]
```

Comment:
needed?

Reply:
yes, otherwise there could be problem when we suggest blocks without genesis block set